backend/src/API/teamEventsAPI.ts

```diff
@@ -1,5 +1,4 @@
 import TeamEventsDao from '../dao/TeamEventsDao';
-import { TeamEvent } from '../types/DataTypes';
```

Comment:
For some reason the backend was still using a separate declaration of the `TeamEvent` type